motr/examples/example1.c

```diff
@@ -45,6 +46,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <errno.h>
+#include "motr/idx.h"
```

Comment:
Can you please show me how you compiled this file?

I did not see such error you mentioned in the commit message.
"motr/idx.h" in included in "motr/client.h". So, "motr/client.h" is the only header file a Motr app needs.

Can you confirm?

Reply:
I add this example1.c manually and compile it while using the "dev" branch, perhaps that's why I got an error. The current main branch is broken, I hope this [PR213](https://github.com/Seagate/cortx-motr/pull/213) will be merged soon.

Reply:
Please don't use "dev" branch any more. It is obsolete. It is not maintained and will not be updated.

Along with this example, 
>"motr/idx.h" in included in "motr/client.h". So, "motr/client.h" is the only header file a Motr app needs.
This is to minimize the requirements of header files.

If you work on the "dev" branch, it's OK to include the "motr/idx.h" in source file.
But this is not needed for main branch.